src/components/AddBookForm.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const AddBookForm = () => (
+  <form>
+    <input name="title" placeholder="Book title" />
+    <input name="author" placeholder="Author" />
+    <button type="button" onClick={null}>ADD BOOK</button>
```

Comment:
- [ ] This is a submit button. I suggest using `submit` as the button type since we are actually submitting our book.